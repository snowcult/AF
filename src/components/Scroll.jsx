import React from 'react';
import PropTypes from 'prop-types';
const Scroll = ({ children }) => {
  return (
    <div
      style={{
        overflowY: 'scroll',
        border: '1px solid gray',
      }}
    >
      {children}
    </div>
  );
};
Scroll.propTypes = {
  children: PropTypes.node,
};

export default Scroll;
