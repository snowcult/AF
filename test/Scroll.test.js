import { shallow } from 'enzyme';
import React from 'react';
import Scroll from '../src/components/Scroll.jsx';

it('expect to render Scroll comp', () => {
  const wrapper = shallow(<Scroll />);
  expect(wrapper).toMatchSnapshot();
});
