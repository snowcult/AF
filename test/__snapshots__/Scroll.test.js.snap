// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll comp 1`] = `
<div
  style={
    {
      "border": "1px solid gray",
      "overflowY": "scroll",
    }
  }
/>
`;
